.changeset/brown-kids-do.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
```

Comment:
For this kind of PR is it counter productive to include changesets:
- bumping the app is unnecessary to make the changes work
- there is nothing changing the app user-wise
- adding dev scripts is not something that should appear in changelogs

Reply:
Makes sense, I'll remove them 👍